platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Telemetry.java

```diff
@@ -2,11 +2,12 @@
 
 
 import com.mapbox.android.telemetry.MapboxTelemetry;
+import com.mapbox.android.telemetry.SessionInterval;
 import com.mapbox.android.telemetry.TelemetryEnabler;
 import com.mapbox.mapboxsdk.BuildConfig;
 import com.mapbox.mapboxsdk.Mapbox;
 
-public class Events {
+public class Telemetry {
```

Comment:
Out of curiosity, why did we change the class name?

Reply:
Events doesn't expose the purpose of the class in context of the Maps sdk, it's too generic, Telemetry is clear and straightforward.